src/themes/nautilus/index.js

```diff
@@ -211,6 +220,15 @@ export const theme = {
           strokeWidth: 1.125,
         },
       },
+
+      // TODO: implement these!
+      Tag: {
+        capitalisation: 'uppercase', // uppercase, lowercase, sentence, title
+        colourPosition: 'background', // background or border
+        borderRadius: 0,
+        padding:
+          'toUnits(theme.spacing.padding.xSmall) toUnits(theme.spacing.padding.small)',
```

Comment:
This is a string, not string interpolation, so I have no idea how this is working 😆

Reply:
I'm thinking you want:
```suggestion
          `${toUnits(theme.spacing.padding.xSmall)} ${toUnits(theme.spacing.padding.small)}`,
```

Reply:
It's not! It's just a placeholder. Ultimately I'd like for it to reference the spacing values declared in the theme, but right now we don't have a good way of doing that until I refactor some theme variables, which is part of why I don't want to deal with implementing theme variables just yet.

Reply:
(I'm leaving this as is because toUnits isn't available here and it's really just there as a placeholder for when we actually implement this.)